security/guidelines.rst

```diff
@@ -20,5 +20,7 @@ This is an incomplete and preliminary, not exhaustive security related recommend
   party dependencies. For this reason, the :ref:`Using ConanCenter packages in production environments <devops_consuming_conan_center>` section recommends building your own binaries
   from source and storing those binaries in your own private server, without downloading packages directly from ConanCenter.
   The :ref:`local-recipes-index feature<devops_local_recipes_index>` was designed to help in this process.
+- The ``recipes_only`` field for remotes can be used to avoid downloading binaries from untrusted remotes,
```

Comment:
Downloading recipes from untrusted remotes is a high security risk. I am not sure I'd put this here as a security guideline, it is more like a policy/regulation thing, but you cannot and you never should download even recipes only from a remote that is untrusted